DTS-STN/secure-client-hub
# Update main from dev (May)

### Description

This PR updates main to be in line with the latest changes on dev.

### What to test for/How to test
1. Pull in branch
2. Type `npm i` and `npm run dev`
3. Ensure the application runs as expected.
